docs/basics/faq-troubleshooting.md

```diff
@@ -245,19 +261,25 @@ Using [`risky`](./options.md#matchmode) matching will match torrents for snatchi
 Using [`safe`](./options.md#matchmode) matching will match torrents for snatching based on the size and require that the release groups match and compare them, requiring file structure to match.
 :::
 
-:::caution
+:::danger WARNING
 There are TV shows that are not combined into season packs, such as Jeopardy, Talk Shows, or similar. These shows often have naming which specifies dates, rather than SeasonEpisode style. Searching with `includeSingleEpisodes` and using `risky` on these torrents, or files, can result in _many torrent files being downloaded_ for comparisons for matching only _one individual episode_.
 
 Remember, this would occur on just one (torrent) search, within the entire (all data) search.
 :::
 
 ---
 
-It is highly recommended to use [`excludeOlder` and `excludeRecentSearch`](./daemon.md#set-up-periodic-searches). Both of these are covered [here](./daemon.md#set-up-periodic-searches) as well as on the [options page](./options.md#excludeolder).
+##### Final Notes
+
+:::tip Be ADVISED
+**There is no one-size-fits-all setting for this**, but you can run `cross-seed` with your config and see how many torrents it will search (noted as "suitable" prior to conducting the actual search) and adjust accordingly.
+:::
+
+-   It is highly recommended to use [`excludeOlder` and `excludeRecentSearch`](./daemon.md#set-up-periodic-searches). Both of these are covered [here](./daemon.md#set-up-periodic-searches) as well as on the [options page](./options.md#excludeolder).
 
-Setting something reasonable for both of these based on the amount of content you download is crucial. While we cache your .torrents, it is not beneficial, for example, to repeatedly search the same torrents that are 6 months old every few days. There is no one-size-fits-all setting for this, but you can run `cross-seed` with your config and see how many torrents it will search and adjust accordingly.
+    -   Setting something reasonable for both of these based on the amount of content you download is crucial. While we cache your .torrents, it is not beneficial, for example, to repeatedly search the same torrents that are 6 months old every few days.
```

Comment:
Spacing seems off here compared to the point above and below it.

Reply:
If this is supposed to be a sublist item ignore the comment above.

Reply:
it's supposed to be this way.